addons/website/views/res_config_settings_views.xml

```diff
@@ -105,43 +105,17 @@
-                                <div class="o_setting_left_pane">
-                                </div>
-                                <div class="o_setting_right_pane">
-                                    <label for="auth_signup_uninvited"/>
-                                    <span class="fa fa-lg fa-globe" title="Values set here are website-specific." groups="website.group_multi_website"/>
-                                    <div class="text-muted">
-                                        Let your customers log in to see their documents
-                                    </div>
-                                    <div class="mt8">
-                                        <field name="auth_signup_uninvited" class="o_light_label" widget="radio" required="True"/>
-                                    </div>
-                                    <div class="mt8 content-group">
-                                        <button type="object" name="open_template_user" string="Default Access Rights" icon="fa-arrow-right" class="btn-link"/>
-                                    </div>
-                                </div>
-                            </div>
-                            <div class="col-12 col-lg-6 o_setting_box" id="website_logo_setting">
-                                <div class="o_setting_right_pane">
-                                    <label for="website_logo"/>
-                                    <span class="fa fa-lg fa-globe" title="Values set here are website-specific." groups="website.group_multi_website"/>
-                                    <div class="text-muted">
-                                        Display this logo on the website.
+                                    <div groups="base.group_no_one">
+                                        <div class="mt8 text-muted" title="You can have 2 websites with same domain AND a condition on country group to select wich website use.">
+                                            Once the selection of available websites by domain is done, you can filter by country group.
+                                        </div>
+                                        <div class="mt8">
+                                            <field name="website_country_group_ids" widget="many2many_tags"/>
+                                        </div>
                                     </div>
-                                    <field name="website_logo" widget="image" class="w-25 mt-2"/>
                                 </div>
                             </div>
-                                                        <div class="col-12 col-lg-6 o_setting_box" groups="website.group_multi_website">
+                            <div class="col-12 col-lg-6 o_setting_box" groups="website.group_multi_website">
```

Comment:
Please give this a name or ID so we can easily `xpath` into it.

Reply:
you can //field[@name='specific_user_account']/../..

Reply:
Yeah sure but the long & ugly xpaths break much faster. It is a very minor impact to add it now but a big impact for devs to do cleaner xpaths on this in the future.

Reply:
yes don't worry it is already updated... but not a ~bug ;) 

And not really since each version we change all the settings page, in all case you will need to check your xpath :/